data/packrat/artifacts.json

```diff
@@ -0,0 +1,2052 @@
+    "category": "chats",
+    "file_artifact": [
+      {
+        "filetypes": "database",
+        "path": "AppData",
+        "dir": "ViberPC",
+        "artifact": "viber.db",
+        "description": "All Contact's names, numbers, sms are saved from user's mobile",
+        "credential_type": "database"
+      },
+      {
+        "filetypes": "thumbs",
+        "path": "AppData",
+        "dir": "ViberPC",
+        "artifact": "Thumbs.db",
+        "description": "Viber's Contact's profile images in Thumbs.db file",
+        "credential_type": "image"
+      },
+      {
+        "filetypes": "images",
+        "path": "AppData",
+        "dir": "ViberPC",
+        "artifact": "*.jpg",
+        "description": "Collects all images of contacts and sent recieved",
+        "credential_type": "image"
+      }
+    ]
+  },
+  {
+    "application": "xchat",
+    "category": "chats",
+    "file_artifact": [
+      {
+        "filetypes": "chat_logs",
+        "path": "AppData",
+        "dir": "X-Chat 2",
+        "artifact": "*.txt",
+        "description": "Collects all chatting conversations of sent and recieved",
+        "credential_type": "text",
+        "regex_search": [
+          {
+            "extraction_description": "Searches for credentials (USERNAMES/PASSWORDS)",
+            "extraction_type": "credentials",
+            "regex": [
+              "(?i-mx:password.*)",
+              "(?i-mx:username.*)"
+            ]
+          },
+          {
+            "extraction_description": "searches for Email TO/FROM address",
+            "extraction_type": "Email addresses",
+            "regex": [
+              "(?i-mx:to:.*)",
+              "(?i-mx:from:.*)"
+            ]
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "application": "xfire",
+    "category": "gaming",
+    "file_artifact": [
+      {
+        "filetypes": "logins",
+        "path": "AppDataLocal",
+        "dir": "Xfire",
+        "artifact": "XfireUser.ini",
+        "description": "Xfire saved Username & Passwords",
+        "credential_type": "text",
+        "regex_search": [
+          {
+            "extraction_description": "Searches for Identification",
+            "extraction_type": "Identification",
+            "regex": [
+              "(?i-mx:encrypteduser.*)",
+              "(?i-mx:encryptedlastlogin.*)"
+            ]
+          }
+        ]
+      },
+      {
+        "filetypes": "logins",
+        "path": "AppData",
+        "dir": "Xfire",
+        "artifact": "XfireUser.ini",
+        "description": "Xfire username and password",
+        "credential_type": "text",
+        "regex_search": [
+          {
+            "extraction_description": "Searches for Identification",
+            "extraction_type": "Identification",
+            "regex": [
+              "(?i-mx:encrypteduser.*)",
+              "(?i-mx:encryptedlastlogin.*)"
+            ]
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "application": "chrome",

```

Comment:
possibly a duplicate of https://github.com/rapid7/metasploit-framework/blob/76954957c740525cff2db5a60bcf936b4ee06c42/modules/post/windows/gather/enum_chrome.rb